hw/ip/edn/doc/theory_of_operation.md

```diff
@@ -71,9 +72,10 @@ For details on the options for application interface commands please see the [CS
 Once the CSRNG instance has been instantiated, and the `generate` and `reseed` commands have been loaded, auto request mode can be entered by programming the [`CTRL`](registers.md#ctrl) register with `EDN_ENABLE` and `AUTO_REQ_MODE` fields are enabled.
 Note that if BOOT_REQ_MODE is asserted the state machine will enter boot-time request mode, even if AUTO_REQ_MODE is asserted.
 
-To issue any new commands other than those stored in the generate or reseed FIFOs, it is important to disable auto request mode, by deasserting the `AUTO_REQ_MODE` field in the [`CTRL`](registers.md#ctrl) register.
+To issue any new commands other than those stored in the generate or reseed FIFOs, it is important to disable auto request mode. This is done by deasserting the `AUTO_REQ_MODE` or the `EDN_ENABLE` field in the [`CTRL`](registers.md#ctrl) register.
 Firmware must then wait until the current command is completed by polling the [`MAIN_SM_STATE`](registers.md#main_sm_state) register.
 Once the state machine returns to the `Idle` or `SWPortMode` states, new firmware-driven commands can be passed to the CSRNG via the [`SW_CMD_REQ`](registers.md#sw_cmd_req) register.
+The generate and reseed FIFOs are reset under four circumstances. These circumstances are (a) when the EDN is disabled, (b) when the `SWPortMode` state is entered, (c) when the boot sequence has completed, or (d) when the EDN enters the `Idle` state after it finishes operation in auto mode.
```

Comment:
This change is unrelated to this PR. I think they are already part of another one.

Reply:
You're right, this doesn't belong here. I'll change it in the next push.